src/services/walletService.js

```diff
@@ -0,0 +1,43 @@
+import localforage from 'localforage';
+import { BASE_CURRENCY, USERS_PROFILES } from '../utils/constants';
+
+export const addCurrency = (userProfile, currencyName, amount) => {
```

Comment:
Again described interface (as an object) is better option while reading function call.